src/Umbraco.Core/Models/PublishedContent/Fallback.cs

```diff
@@ -71,5 +71,11 @@ IEnumerator IEnumerable.GetEnumerator()
         {
             return GetEnumerator();
         }
+
+        public const int DisplayFallBackLanguage = 4;
+        /// <summary>
+        /// Gets the fallback to tree ancestors policy.
+        /// </summary>
+        public static Fallback ToDisplayFallBackLanguage => new Fallback(new[] { DisplayFallBackLanguage });
```

Comment:
To keep this consistent it might be useful to rename this `ToDisplayFallbackLanguage` (`Fallback` rather than `FallBack`)

Reply:
Good spot I've done this now.